content/git/concepts/remote/remote.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Remote'
```

Comment:
I think we might want to change the title and the path to the plural which is our usual formatting